pkg/api/validation/validation.go

```diff
@@ -2272,6 +2276,20 @@ func ValidatePodSpec(spec *api.PodSpec, fldPath *field.Path) field.ErrorList {
 		allErrs = append(allErrs, ValidateHostAliases(spec.HostAliases, fldPath.Child("hostAliases"))...)
 	}
 
+	if len(spec.PriorityClassName) > 0 {
+		if !utilfeature.DefaultFeatureGate.Enabled(features.PodPriority) {
```

Comment:
yes; my version is for sometimes removing this feature gate (just remove the `else` block is OK); both are ok to me :).